docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -29,13 +29,13 @@ You can find all the files needed for this tutorial here: [neb_tutorial.tar.gz](
 ## Setting up the structures
 
 ### Setting up the initial and final states
-These are the local minima structures you want to find the barrier between. They can be equivalent structures with e.g. a translation between them or they can be configurations with different energies (in which case your forward and reverse barriers will not be the same!). There are two crucial things to set up **before running the NEB** though:
+These are the local minima structures you want to find the barrier between. They can be equivalent structures, e.g. differing by a symmetry operation, such reflection through a mirror plane, or they can be configurations with different energies (in which case your forward and reverse barriers will not be the same!). Two crucial conditions must be met **before running the NEB calculation**:
 
-1. The end point structures must be fully relaxed (geometry optimised) before you start the NEB.
-2. The atoms must be in the correct order. A very common mistake is to use some third party tool to generate the final state from the initial state and end up with atoms that don't match the initial atom order. When CASTEP then goes to interpolate between the two structures, you end up with atoms "passing through each other", causing the calculation to blow up. Always check that the atoms connect up in the way you expect. 
+1. The end point structures *[and any intermediates?]* must be fully relaxed (geometry optimised) *[using the same parameters as NEB presumably]* before you start the NEB.
```

Comment:
You typically generate the intermediate (guess) structure based the (optimised) end points (e.g. through linear interpolation). The intermediate structure will be relaxed during the course of the NEB, so it doesn't need to be optimised beforehand. In fact, if you tried to optimise it independently, it might well roll down to one of the end point structures.

Reply:
Yes, I realised afterwards that you can't optimise the intermediate! I think it's useful to state this explicitly in a tutorial context. I would also state the point about consistent calculations. It's obvious to an expert, but again, you can be explicit in a tutorial.

Reply:
```suggestion
1. The end point structures must be fully relaxed (geometry optimised) before you start the NEB. Note that both the initial relaxations and the actual NEB calculation must use consistent parameters (e.g. basis set, pseudopotentials, k-points, XC functional etc.).
```